tests/e2e-ta-collector-mtls/ta-collector-mtls/chainsaw-test.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: targetallocator-collector-mtls
+spec:
+  steps:
+  - name: step-00
+    try:
+    - apply:
+        template: true
+        file: 00-install.yaml
+    - assert:
+        file: 00-assert.yaml
```

Comment:
Looks like you need to add your other two steps here.

Reply:
Waiting for https://github.com/open-telemetry/opentelemetry-operator/pull/3120

Reply:
It was merged, can we add them now?